lib/widgets/dialog.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+void showErrorDialog({required BuildContext context, required String title, String? message}) {
+  showDialog(
+      context: context,
+      builder: (BuildContext context) => AlertDialog(
+        // TODO(i18n)
+        title: Text(title),
+
+        // TODO(i18n)
+        content: message != null ? Text(message) : null,
```

Comment:
The docs for this property say:
> Typically this is a SingleChildScrollView that contains the dialog's message. As noted in the AlertDialog documentation, it's important to use a SingleChildScrollView if there's any risk that the content will not fit.

Seems like there is such a risk here — the message could be several lines long, and the device could be short or in landscape mode or be set to a giant font size — so we should do that.